### DISCUSSION TITLE
Route [platform.index] not defined.

### DISCUSSION THREAD

**Discussion by nettsite** [orchidsoftware/platform#2251] [Q&A] [answered]
Hi,

I am getting this error on a clean Orchid platform install: 

`[Route [platform.index] not defined.]`

I can see many places in the source where 'platform.index' is specified as a route, but nowhere that it is created.

`<a class="header-brand order-last" href="{{route('platform.index')}}">  @includeFirst([config('platform.template.header'), pllatform::header'])
                </a>`

That is from 
`vendor / orchid / platform / resources / views / dashboard.blade.php : 17`

What am I doing wrong?

**Comment by gustavoconter** [comment]
The route should be defined at routes/platform.php.
Check if a route with the name "platform.index" exists inside that file.

**Reply by nettsite** [reply]
That is what I would have thought - and it it was the first place I looked, but it is not defined there. This is a new, unaltered, clean install.

It is mentioned as a parent for breadcrumbs, in platform.php:
`
\Route::screen('profile', UserProfileScreen::class)
    ->name('platform.profile')
    ->breadcrumbs(function (Trail $trail) {
        return $trail
            ->parent('platform.index')
            ->push(__('Profile'), route('platform.profile'));
    });
`

**Comment by nettsite** [comment, marked as answer]
Seems this is the trick:

You need to remove the default route from web.php:
`
// Route::get('/', function () {
//     return view('welcome');
// });
`

**Reply by Oxicode** [reply]
u ar awsome bro!!!

**Comment by esmultiservicios** [comment]
this works to me

changing this 

<?php

use Illuminate\Support\Facades\Auth;
use Illuminate\Support\Facades\Route;

Route::get('/', function () {
     return view('welcome');
});



to

<?php

use Illuminate\Support\Facades\Auth;
use Illuminate\Support\Facades\Route;

Route::get('/', function () {
    // Redirigir solo si el usuario no está autenticado
    return Auth::check() ? redirect('/admin/main') : redirect('/admin');
});
